server/world/viewer.go

```diff
@@ -47,6 +47,8 @@ type Viewer interface {
 	// ViewEntityState views the current state of an entity. It is called whenever an entity changes its
 	// physical appearance, for example when sprinting.
 	ViewEntityState(e Entity)
+	// ViewEntityAnimation views an animation performed by an entity.
```

Comment:
I would probably change this to mention that it triggers an animation to start playing instead since it currently sounds like it views an already-playing animation